x/icactl/module_ibc.go

```diff
@@ -36,13 +38,17 @@ func (am IBCModule) OnChanOpenInit(
 	chanCap *capabilitytypes.Capability,
 	counterparty channeltypes.Counterparty,
 	version string,
-) error {
+) (string, error) {
+	if strings.TrimSpace(version) == "" {
+		version = icatypes.Version
+	}
+
```

Comment:
don't know who will call `OnChanOpenInit`,  feels like the version can be checked in the caller function, except the caller doesn't have the dependency of `icatypes`

Reply:
it's a generic interface called by ibc core modules